### ISSUE TITLE
cargo.lock should read cardo.toml in warning when running tests?

### ISSUE THREAD

**Issue by TheDaysy** [exercism/rust-test-runner#45]
When submitting an exercise using an external crate for the first time I received the warning
```
WARNING: student did not upload Cargo.lock. This may cause build errors.
```
and got an `unresolved import` error, so I resubmitted including `cargo.lock` and still got the same unresolved import error.
after a few attempts and requesting mentoring I was told that I should in fact be submitting `cargo.toml` which i did and the tests then ran.

I am still very new to Rust and to Exercism so wanted to post an issue before just putting in a pull request to check that this is correct and that my experience was not an outlier.

**Comment by coriolinus** [comment]
Yes, the test runner should be updated. That warning was created before external crates were supported at all (https://github.com/exercism/rust-test-runner/pull/27). Also note that `Cargo.toml` wasn't submitted by default through the web interface until https://github.com/exercism/rust/pull/1459.

A PR updating or removing that warning would be welcome.
